python/tests/unit/arcticdb/version_store/test_engine.py

```diff
@@ -14,32 +14,37 @@
 import random
 import sys
 from arcticdb.version_store import TimeFrame
+from tests.util.mark import random_seed_context
 
 
 def gen_params():
-    # colnums
-    p = [list(range(2, 5))]
-    # rownums
-    periods = 6
-    p.append(list(range(1, periods + 2)))
-    # cols
-    p.append(list(chain(*[list(combinations(["a", "b", "c"], c)) for c in range(1, 4)])))
-    # tsbounds
-    p.append([(j, i) for i in range(periods) for j in range(i)])
-    return random.sample(list(product(*p)), 1000)
+    with random_seed_context():
```

Comment:
Sorry for not going into this detail in my previous comment, but you can use `@contextmanager` functions as decorators:

```python
@random_seed_context
def gen_params():
...
```

should work

Reply:
Yes, I know but I had some problem with it being a decorator and also I like this approach better.